RecoLocalTracker/SiPixelRecHits/plugins/PixelRecHits.cc

```diff
@@ -0,0 +1,65 @@
+// C++ headers
+#include <algorithm>
+#include <numeric>
+
+// CUDA runtime
+#include <cuda_runtime.h>
+
+// CMSSW headers
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/launch.h"
+#include "RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.h"
+#include "RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusteringConstants.h"
+
+#include "PixelRecHits.h"
+#include "gpuPixelRecHits.h"
```

Comment:
still, do not understand how, w/o cudaCompat that should NOT supposed to be active in this case, gcc would swallow cuda syntax.
Do you have self-contained trivial unit test that demonstrates how to launch a kernel from plain c++  compiled with gcc?